card/mod/account/set/right/account.rb

```diff
@@ -174,8 +174,8 @@ def token_path_opts
     # Problem: when you do that then the fields are missing in the sign up form:
     # output( [field_nest(:email, view: :titled, title: "email"),
     #          field_nest(:password, view: :titled, title: "password")])
-    %({{+#{:email.cardname}|edit_row;title:email}}
-      {{+#{:password.cardname}|edit_row;title:password}})
+    %({{+#{:email.cardname}|labeled;title:email;edit:content_inline}}
+      {{+#{:password.cardname}|labeled;title:password;edit:content_inline}})
```

Comment:
Did you check if this looks good? edit_row uses different col classes than labeled.

Reply:
I did.  It's presentable, but as a general rule I think it would look better if only the content area (and not the whole slot) were replaced with the editor and the editor had a border that made it look more like an editor.  Will make a story...